src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.RenameTrackingCodeAction.cs

```diff
@@ -18,13 +18,13 @@ internal sealed partial class RenameTrackingTaggerProvider
     {
         private class RenameTrackingCodeAction : CodeAction
         {
-            private readonly string _title;
+            private readonly LocalizableString _title;
             private readonly Document _document;
             private readonly IEnumerable<IRefactorNotifyService> _refactorNotifyServices;
             private readonly bool _showPreview;
             private readonly ITextUndoHistoryRegistry _undoHistoryRegistry;
 
-            public RenameTrackingCodeAction(Document document, string title, IEnumerable<IRefactorNotifyService> refactorNotifyServices, ITextUndoHistoryRegistry undoHistoryRegistry, bool showPreview)
+            public RenameTrackingCodeAction(Document document, LocalizableString title, IEnumerable<IRefactorNotifyService> refactorNotifyServices, ITextUndoHistoryRegistry undoHistoryRegistry, bool showPreview)
```

Comment:
Have we modified the call sites of this constructor to actually pass in a localizable string?